app/helpers/account_helper.rb

```diff
@@ -0,0 +1,13 @@
+module AccountHelper
+  def resource_name
+    :user
+  end
+
+  def resource
+    @resource = current_user
+  end
+
+  def devise_mapping
```

Comment:
Is this being used anywhere?

Reply:
The views wouldn't get the resource with out it